build/azure-pipelines/common/computeNodeModulesCacheKey.js

```diff
@@ -0,0 +1,43 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
Looks like you're building this directory in the pipelines anyways, so any reason to have this checked in?

Reply:
I don't think I have built anything until this step. Atleast for the compile job.